doc/source/changes/version_0_28.rst.inc

```diff
@@ -21,6 +21,9 @@
 New features
 ------------
 
+* made it possible to run the tutorial as a Jupyter notebook by clicking on the ``launch|binder`` badge on
```

Comment:
* s/as a Jupyter notebook/online (as a Jupyter notebook)/?
* s/the top/top/?
* s/HTML page/web page/?